### DISCUSSION TITLE
为啥把联机按钮删了，难不成不做了？

### DISCUSSION THREAD

**Discussion by sdtthdsr7u** [Meloong-Git/PCL#1559] [提问] [answered]
为啥把联机按钮删了，难不成不做了？

**Comment by WForst-Breeze** [comment, marked as answer]
由于当前联机高度商业化，PCL2 决定与 HMCL、BakaXL 联合制作新联机核心，关于此问题具体请见，**[PCL2 / HMCL / BakaXL 对联机功能下一步发展的联合公告](https://www.bilibili.com/read/cv19553725)**

**Reply by sdtthdsr7u** [reply]
OK，我的问题
